src/Syntax.jl

```diff
@@ -158,6 +159,48 @@ macro stock_and_flow(block)
     end
 end
 
+abstract type StockArgT end
```

Comment:
Would it make any sense to combine `StockArgT` and `ParamArgT` into a single `ArgT`? Below they seem to share the same fields for [StockArgUnit, ParamArgUnit]-Symbol/-Expr versions.

Reply:
We could, or maybe have a supertype for both of them.